### DISCUSSION TITLE
ResNet50_dcn_asf_synthtext_pretrained 预训练模型的下载链接

### DISCUSSION THREAD

**Discussion by wangjiannan98** [PaddlePaddle/PaddleOCR#13761] [Q&A]
请问一下 ResNet50_dcn_asf_synthtext_pretrained 预训练模型的下载链接怎么获取，一直没有找到这个预训练模型

**Comment by GreatV** [comment]
- https://github.com/PaddlePaddle/PaddleOCR/issues/7354

**Reply by wangjiannan98** [reply]
您好，这不是一个问题哈，我这边是没有找到ResNet50_dcn_asf_synthtext_pretrained的下载链接

**Reply by wangjiannan98** [reply]
或者paddle是否有预训练模型的集合链接

**Reply by GreatV** [reply]
没有提供的就没有了

**Comment by liuwake** [comment]
download ResNet50_dcn_asf_synthtext_pretrained @ https://paddleocr.bj.bcebos.com/dygraph_v2.1/en_det/ResNet50_dcn_asf_synthtext_pretrained.pdparams .
You can find the link @`PaddleOCR/docs/version2.x/algorithm/text_detection/algorithm_det_db.md`->`[合成数据预训练模型]`
